2020-03-12 19:13:20 WARNING (Recorder) [homeassistant.components.recorder] Ended unfinished session (id=60 from 2020-03-11 19:17:56.985615)
2020-03-12 19:13:25 WARNING (MainThread) [homeassistant.setup] Setup of recorder is taking over 10 seconds.
2020-03-12 19:13:25 WARNING (MainThread) [homeassistant.setup] Setup of frontend is taking over 10 seconds.
2020-03-12 19:13:36 ERROR (MainThread) [homeassistant.components.hue] Error connecting to the Hue bridge at 192.168.178.28
2020-03-12 19:13:36 ERROR (MainThread) [homeassistant.components.hue] Error connecting to the Hue bridge at 192.168.178.28. Retrying in 2 seconds
2020-03-12 19:13:40 WARNING (MainThread) [homeassistant.setup] Setup of group is taking over 10 seconds.
2020-03-12 19:13:40 WARNING (MainThread) [homeassistant.setup] Setup of homekit is taking over 10 seconds.
2020-03-12 19:13:40 WARNING (MainThread) [homeassistant.components.sensor] Setup of platform yr is taking over 10 seconds.
2020-03-12 19:13:42 ERROR (MainThread) [homeassistant.components.device_tracker] Unable to load /home/homeassistant/.homeassistant/known_devices.yaml: Config file not found: /home/homeassistant/.homeassistant/known_devices.yaml
2020-03-12 19:13:50 ERROR (Thread-6) [homeassistant.components.light.yeelight] Failed to connect to bulb 192.168.178.24, Bedroom Lamp: A socket error occurred when sending the command.
2020-03-12 19:13:52 WARNING (MainThread) [homeassistant.components.light] Setup of platform yeelight is taking over 10 seconds.
2020-03-12 19:13:52 WARNING (MainThread) [homeassistant.components.switch] Setup of platform broadlink is taking over 10 seconds.
2020-03-12 19:13:53 ERROR (Thread-5) [homeassistant.components.switch.broadlink] Failed to connect to device
2020-03-12 19:13:57 ERROR (Thread-2) [homeassistant.components.light.yeelight] Failed to connect to bulb 192.168.178.25, Bathroom Ceiling: A socket error occurred when sending the command.
2020-03-12 19:14:28 ERROR (Thread-4) [pychromecast.socket_client] Failed to connect, retrying in 5.0s
2020-03-12 19:17:13 WARNING (Thread-10) [pychromecast.socket_client] Heartbeat timeout, resetting connection
2020-03-12 19:17:24 ERROR (Thread-10) [pychromecast.socket_client] Failed to connect, retrying in 5.0s
2020-03-12 19:37:15 WARNING (Thread-11) [pychromecast.socket_client] Heartbeat timeout, resetting connection
2020-03-12 19:37:45 ERROR (Thread-11) [pychromecast.socket_client] Failed to connect, retrying in 5.0s
2020-03-12 19:37:52 WARNING (Thread-8) [pychromecast.socket_client] Heartbeat timeout, resetting connection
2020-03-12 19:37:55 WARNING (Thread-7) [pychromecast.socket_client] Heartbeat timeout, resetting connection
2020-03-12 19:38:22 ERROR (Thread-8) [pychromecast.socket_client] Failed to connect, retrying in 5.0s
2020-03-12 19:38:25 WARNING (Thread-9) [pychromecast.socket_client] Heartbeat timeout, resetting connection
2020-03-12 19:38:25 ERROR (Thread-7) [pychromecast.socket_client] Failed to connect, retrying in 5.0s
2020-03-12 19:38:55 ERROR (Thread-9) [pychromecast.socket_client] Failed to connect, retrying in 5.0s
2020-03-12 19:56:21 ERROR (Thread-8) [pychromecast.socket_client] Failed to connect: [Errno 113] No route to host. aborting due to stop signal.
2020-03-12 19:59:42 WARNING (Thread-13) [pychromecast.socket_client] Heartbeat timeout, resetting connection
2020-03-12 19:59:48 WARNING (Thread-12) [pychromecast.socket_client] Heartbeat timeout, resetting connection
2020-03-12 20:00:12 ERROR (Thread-13) [pychromecast.socket_client] Failed to connect, retrying in 5.0s
2020-03-12 20:00:18 ERROR (Thread-12) [pychromecast.socket_client] Failed to connect, retrying in 5.0s
2020-03-12 21:33:26 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-12 22:13:40 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-12 23:09:57 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-13 07:24:34 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-13 07:27:39 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-13 09:44:55 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 (Error requesting data from 192.168.178.28: Cannot connect to host 192.168.178.28:80 ssl:None [Network is unreachable])
2020-03-13 09:44:57 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-13 09:45:00 WARNING (zeroconf-ServiceBrowser__googlecast._tcp.local.) [zeroconf] Exception occurred:
Traceback (most recent call last):
  File "/srv/homeassistant/lib/python3.5/site-packages/zeroconf.py", line 2101, in send
    bytes_sent = s.sendto(packet, 0, (addr, port))
OSError: [Errno 22] Invalid argument
2020-03-13 11:06:28 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-13 11:18:19 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-13 12:15:58 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-13 13:07:45 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-13 13:38:12 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-13 14:18:24 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-13 14:39:35 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-13 16:17:40 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-13 17:08:49 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-13 17:11:24 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-13 17:35:40 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-13 18:18:32 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-13 18:26:17 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-13 19:29:18 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-13 20:00:49 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-13 20:58:09 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-13 22:21:19 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-13 22:40:25 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-13 23:01:03 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-13 23:03:07 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-14 04:15:28 ERROR (MainThread) [homeassistant.components.sensor.yr] Retrying in 20 minutes: 
2020-03-14 04:35:39 ERROR (MainThread) [homeassistant.components.sensor.yr] Retrying in 16 minutes: 
2020-03-14 08:50:39 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-14 10:07:52 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-14 11:33:19 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-14 11:38:29 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-14 12:41:56 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-14 15:28:40 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-14 16:09:29 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-14 16:34:48 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-14 17:52:43 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-14 17:55:15 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-14 17:58:27 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-14 18:05:07 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-14 18:19:03 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-14 20:04:19 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-14 21:17:06 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-14 23:01:22 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-15 00:12:06 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-15 01:58:55 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-15 02:23:00 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-15 03:06:12 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-15 10:37:11 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-15 11:22:38 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-15 11:28:19 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-15 11:45:17 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-15 13:01:13 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-15 13:16:38 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-15 13:33:11 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-15 13:35:12 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-15 13:35:18 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-15 13:44:28 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-15 13:45:10 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-15 14:19:33 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-15 15:12:16 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-15 15:38:32 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-15 17:18:11 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-15 18:41:49 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-15 18:45:08 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-15 19:31:49 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-15 20:09:04 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-15 20:53:59 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-15 22:04:45 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-15 23:25:19 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-15 23:26:52 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-15 23:29:27 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-16 00:09:44 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-16 00:48:45 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-16 00:48:56 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-16 01:33:48 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-16 02:02:43 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-16 02:03:14 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-16 04:19:45 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-16 04:20:04 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-16 04:24:02 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-16 04:24:10 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-16 04:54:19 ERROR (MainThread) [homeassistant.components.sensor.yr] Retrying in 15 minutes: 
2020-03-16 05:09:30 ERROR (MainThread) [homeassistant.components.sensor.yr] Retrying in 18 minutes: 
2020-03-16 05:13:38 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-16 10:09:40 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-16 10:13:52 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 (Error requesting data from 192.168.178.28: Cannot connect to host 192.168.178.28:80 ssl:None [Connect call failed ('192.168.178.28', 80)])
2020-03-16 10:34:06 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-16 11:11:14 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-16 11:58:09 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-16 12:23:58 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-16 13:28:56 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-16 13:33:34 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-16 15:15:43 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-16 16:12:27 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-16 16:46:02 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-16 17:10:16 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-16 17:39:11 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-16 17:59:20 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-16 17:59:49 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to turn the bulb off: Bulb closed the connection.
2020-03-16 17:59:54 ERROR (MainThread) [homeassistant.helpers.entity] Update for light.bedroom_lamp fails
Traceback (most recent call last):
  File "/srv/homeassistant/lib/python3.5/site-packages/yeelight/main.py", line 360, in send_command
    data = self._socket.recv(16 * 1024)
OSError: [Errno 9] Bad file descriptor

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/srv/homeassistant/lib/python3.5/site-packages/homeassistant/helpers/entity.py", line 221, in async_update_ha_state
    await self.async_device_update()
  File "/srv/homeassistant/lib/python3.5/site-packages/homeassistant/helpers/entity.py", line 349, in async_device_update
    await self.hass.async_add_executor_job(self.update)
  File "/usr/lib/python3.5/asyncio/futures.py", line 380, in __iter__
    yield self  # This tells Task to wait for completion.
  File "/usr/lib/python3.5/asyncio/tasks.py", line 304, in _wakeup
    future.result()
  File "/usr/lib/python3.5/asyncio/futures.py", line 293, in result
    raise self._exception
  File "/usr/lib/python3.5/concurrent/futures/thread.py", line 55, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/srv/homeassistant/lib/python3.5/site-packages/homeassistant/components/light/yeelight.py", line 314, in update
    self._bulb.get_properties()
  File "/srv/homeassistant/lib/python3.5/site-packages/yeelight/main.py", line 321, in get_properties
    response = self.send_command("get_prop", requested_properties)
  File "/srv/homeassistant/lib/python3.5/site-packages/yeelight/main.py", line 363, in send_command
    self.__socket.close()
AttributeError: 'NoneType' object has no attribute 'close'
2020-03-16 18:27:13 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-16 20:24:24 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-16 20:25:15 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-16 20:25:20 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: A socket error occurred when sending the command.
2020-03-16 20:25:25 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-16 20:34:42 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-16 20:48:07 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-16 21:29:28 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-16 21:57:21 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-16 23:52:23 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-17 00:02:17 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-17 00:39:59 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-17 00:55:29 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-17 02:42:50 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-17 09:53:10 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-17 10:44:11 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-17 12:04:11 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-17 13:59:17 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-17 14:34:55 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-17 15:31:44 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-17 16:20:46 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-17 17:14:30 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-17 17:42:52 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-17 18:30:56 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-17 18:37:05 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-17 19:34:52 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-17 20:33:39 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-17 20:48:12 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-17 22:28:23 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-17 22:45:24 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-17 23:38:00 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-17 23:58:09 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-18 00:23:58 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-18 00:43:39 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-18 01:47:04 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-18 01:49:08 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-18 02:27:53 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-18 02:31:29 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-18 10:01:30 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-18 11:00:19 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-18 11:31:18 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-18 13:30:23 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-18 13:41:45 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-18 15:43:37 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-18 16:55:32 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-18 17:50:28 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-18 18:42:40 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-18 18:45:09 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-18 19:37:19 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-18 19:46:06 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-18 19:51:38 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-18 20:16:32 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-18 21:17:27 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-18 21:41:26 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-18 21:41:38 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-18 23:33:09 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-19 00:48:04 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-19 02:39:59 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-19 02:42:00 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-19 02:59:38 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-19 03:14:51 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-19 03:31:29 ERROR (MainThread) [homeassistant.components.sensor.yr] Retrying in 20 minutes: 
2020-03-19 03:51:40 ERROR (MainThread) [homeassistant.components.sensor.yr] Retrying in 19 minutes: 
2020-03-19 04:10:51 ERROR (MainThread) [homeassistant.components.sensor.yr] Retrying in 20 minutes: 
2020-03-19 04:31:02 ERROR (MainThread) [homeassistant.components.sensor.yr] Retrying in 16 minutes: 
2020-03-19 04:47:13 ERROR (MainThread) [homeassistant.components.sensor.yr] Retrying in 18 minutes: 
2020-03-19 08:52:41 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-19 08:55:14 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-19 11:55:51 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-19 12:14:53 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-19 13:42:14 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-19 13:59:16 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-19 14:44:40 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-19 15:27:31 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-19 16:04:10 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-19 16:05:08 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-19 16:52:40 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-19 17:18:55 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-19 17:45:18 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-19 19:43:26 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-19 19:55:45 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-19 21:57:26 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-19 21:59:53 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-19 22:01:00 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-19 22:05:13 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-19 23:42:05 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-20 00:39:16 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-20 01:40:37 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-20 10:03:54 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-20 11:02:07 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-20 12:24:44 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-20 12:25:09 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-20 12:25:15 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-20 13:46:45 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-20 14:34:15 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-20 14:35:02 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-20 16:38:13 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-20 17:56:33 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-20 19:05:45 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-20 19:09:22 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-20 19:30:00 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-20 20:16:56 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-20 20:53:37 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-20 21:29:56 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-20 22:09:32 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-20 22:43:31 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-21 00:28:18 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-21 01:11:40 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-21 01:15:07 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-21 01:43:13 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-21 01:45:11 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-21 02:29:34 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-21 03:17:04 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-21 03:19:07 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-21 04:09:43 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-21 12:30:18 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-21 13:39:26 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-21 16:30:40 ERROR (Thread-6) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-21 18:21:09 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-21 18:45:10 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-21 18:45:23 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-21 18:49:31 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-21 19:22:28 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-21 19:25:13 ERROR (MainThread) [homeassistant.components.light.hue] Unable to reach bridge 192.168.178.28 ()
2020-03-21 20:14:00 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-21 20:47:02 ERROR (Thread-4) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-21 21:24:12 ERROR (Thread-5) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
2020-03-21 21:48:21 ERROR (Thread-3) [homeassistant.components.light.yeelight] Unable to update bulb status: Bulb closed the connection.
